MyKkumi/MyKkumi/Domain/Model/HelloWorld.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  File.swift
+//  MyKkumi
+//
+//  Created by 최재혁 on 6/17/24.
+//
+
+import Foundation
+
+public struct HelloWorld : Codable {
+    let title : String?
+    
+    enum CodingKeys : String, CodingKey {
+        case title = "title"
```

Comment:
변수명과 CodingKey(API Respoonse JSON 필드명) 이 같은 경우 

```suggestion
        case title
```

이렇게만 작성하셔도 됩니다. 다른 경우에만 따로 작성합니다.